src/js/constants/placeholders.ts

```diff
@@ -8,5 +8,6 @@ export const PLACEHOLDERS = {
   [SEARCH_TYPES.ROMANIZED]: ['jo mange thakur apne te soi', true], // romanized
   [SEARCH_TYPES.ANG]: ['123', true], // ang
   [SEARCH_TYPES.MAIN_LETTERS]: ['j mgh Tkr Apn q'], // main letters
-  [SEARCH_TYPES.ROMANIZED_FIRST_LETTERS_ANYWHERE]: ['jmtatssd', true] //romanized first letter anywhere
+  [SEARCH_TYPES.ROMANIZED_FIRST_LETTERS_ANYWHERE]: ['jmtatssd', true], //romanized first letter anywhere
+  [SEARCH_TYPES.ASK_A_QUESTION]: ['What is the best time to meditate', true] // ask a question
```

Comment:
Should we add a question mark at the end?

Reply:
Yes, added.